### PR TITLE
RUNTIME-3388 initial changes

### PR DESCRIPTION
**Pull request by jkonrath-postman**
* image change
* split up the parameters to two tabs
* add advanced options parameters

### CODE REVIEW

File: src/pages/docs/sending-requests/authorization.md [hunk 1]
```diff
@@ -320,6 +322,11 @@ The full list of parameters to request a new access token is as follows, dependi
 * __State:__ An opaque value to prevent cross-site request forgery.
 * __Client Authentication:__ A dropdown—send a Basic Auth request in the header, or client credentials in the request body. _After upgrading to a new version, change the value in this dropdown menu to avoid problems with client authentication._
 
+#### Advanced Options tab
+
+* __Resource:__ A URI that indicates the resource or target service where the token is intended to be used.
+* __Audience:__ A URI that target audience or service where the token is intended to be used.
+
 When your config is complete, click __Request Token__.
```
Comment (giridharvc7) [inline src/pages/docs/sending-requests/authorization.md:330]: I don't see __Request Token__ in the product, do we mean __Get New Access Token__

**Review by giridharvc7 — CHANGES_REQUESTED** [review]
Just one comment, otherwise LGTM

File: src/pages/docs/sending-requests/authorization.md [hunk 2]
```diff
@@ -320,7 +322,12 @@ The full list of parameters to request a new access token is as follows, dependi
 * __State:__ An opaque value to prevent cross-site request forgery.
 * __Client Authentication:__ A dropdown—send a Basic Auth request in the header, or client credentials in the request body. _After upgrading to a new version, change the value in this dropdown menu to avoid problems with client authentication._
 
-When your config is complete, click __Request Token__.
+#### Advanced Options tab
+
+* __Resource:__ A URI that indicates the resource or target service where the token is intended to be used.
+* __Audience:__ A URI that target audience or service where the token is intended to be used.
```
Comment (claireriley) [inline src/pages/docs/sending-requests/authorization.md:328]:
```suggestion
* __Audience:__ A URI that indicates the target audience or service where the token is intended to be used.
```

File: src/pages/docs/sending-requests/authorization.md [hunk 3]
```diff
@@ -306,7 +306,9 @@ Enter the provider's __Access Token URL__, together with the __Client ID__ and _
 
 ### Requesting an OAuth 2.0 token
 
-The full list of parameters to request a new access token is as follows, depending on your grant type:
+The full list of parameters to request a new access token is as follows, depending on your grant type
+
+#### Configuration Options tab
```
Comment (claireriley) [inline src/pages/docs/sending-requests/authorization.md:311]: Add to TOC at top of page along with `#### Advanced Options tab`

File: src/pages/docs/sending-requests/authorization.md [hunk 4]
```diff
@@ -306,7 +306,9 @@ Enter the provider's __Access Token URL__, together with the __Client ID__ and _
 
 ### Requesting an OAuth 2.0 token
 
-The full list of parameters to request a new access token is as follows, depending on your grant type:
+The full list of parameters to request a new access token is as follows, depending on your grant type
```
Comment (claireriley) [inline src/pages/docs/sending-requests/authorization.md:309]: Could add a period at the end of the line, I'm unsure if either option is more appropriate

**Review by claireriley — APPROVED** [review]
Looks good! Added a few comments, unsure if they are blockers so approving

**Pull request merged**